public/css/app.css

```diff
@@ -143,18 +140,27 @@ input {
   width: 100%;
 }
 
-input::placeholder,
-textarea::placeholder {
-  color: #000;
+input {
```

Comment:
```
✖  Unexpected duplicate selector "input", first used at line 99     no-duplicate-selectors
```

In fact, once I resolved the ESLint errors locally, I got a small handful(s) of CSS warnings/errors:

```
public/css/app.css
 143:1   ✖  Unexpected duplicate selector "input", first used at line 99     no-duplicate-selectors
 148:18  ✖  Expected single space before "{"                                 block-opening-brace-space-before
 157:1   ✖  Expected no more than 1 empty line                               max-empty-lines
 426:29  ✖  Expected single space before "{"                                 block-opening-brace-space-before
 647:1   ✖  Expected no more than 1 empty line                               max-empty-lines
 686:3   ✖  Unexpected duplicate "display"                                   declaration-block-no-duplicate-properties
 719:15  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after
 749:1   ✖  Unexpected empty line before closing brace                       block-closing-brace-empty-line-before
 767:13  ✖  Expected single space after ":" with a single-line declaration   declaration-colon-space-after
 778:1   ✖  Unexpected empty line before closing brace                       block-closing-brace-empty-line-before
 786:1   ✖  Expected no more than 1 empty line                               max-empty-lines
 787:1   ✖  Expected no more than 1 empty line                               max-empty-lines
```